frontend/src/metabase/entities/users.js

```diff
@@ -29,6 +29,10 @@ const Users = createEntity({
 
   path: "/api/user",
 
+  objectSelectors: {
+    getName: user => user.common_name,
+  },
```

Comment:
This apparently is used in 
https://github.com/metabase/metabase/blob/8472a9917b87a9e8ba5e8de2d2fe037dd1f7e58e/frontend/src/metabase/entities/containers/EntityName.jsx#L14

e.g. at the header of the admin > audit > team members > all members > team member detail
![image](https://user-images.githubusercontent.com/1937582/174002607-5ed7e2c0-0ca7-4d36-ae38-5533882a4703.png)